.builder/actions/build_samples.py

```diff
@@ -28,6 +34,9 @@ def run(self, env):
                           f'-H{sample_path}',
                           f'-DCMAKE_PREFIX_PATH={env.install_dir}',
                           '-DCMAKE_BUILD_TYPE=RelWithDebInfo'])
+            # append extra cmake configs
+            for cmake_step in cmd_args.cmake_extra:
+                steps[-1].append(cmake_step)
```

Comment:
style/trivial: python has the `extend()` method for gluing lists together. this could be like:

`steps[-1].extend(cmd_args.cmake_extra)`